jobs/cronjob-ldap-group-sync.yml

```diff
@@ -62,65 +82,71 @@ objects:
                 command:
                   - "/bin/bash"
                   - "-c"
-                  - "oc adm groups sync --sync-config=/etc/config/ldap-group-sync.yaml --confirm || :"
+                  - "oc adm groups sync --sync-config=/etc/config/ldap-group-sync.yaml --confirm"
```

Comment:
This needs to be re-added - i.e.: the ` || : `. Please see this PR for the reason behind this:
https://github.com/redhat-cop/openshift-management/pull/24/files

Reply:
Hrm... I want to research this more. This doesn't seem like an appropriate fix as it would mask when ldap group sync is indeed failing. It seems the correct solution is to set `spec.backoffLimit` to 0 on the Job (cronjob's `spec.template.spec.backoffLimit`). I've tested this with OCP4 and it works as expected. Still need to check with 3.11.

We may not have had the option of using the backoffLimit when this PR was opened last April.